Controllers/OperationController.cs

```diff
@@ -16,18 +17,45 @@ namespace ARMExplorer.Controllers
     public class OperationController : ApiController
     {
         private readonly IArmRepository _armRepository;
+        private static readonly MemoryCache SwaggerCache = new MemoryCache("SwaggerDefinitionCache");
```

Comment:
Sure, a good comment will help, as  I really struggled to understand that line :)